frontend/interfaces/host.ts

```diff
@@ -60,6 +62,12 @@ export default PropTypes.shape({
   users: PropTypes.arrayOf(hostUserInterface),
   policies: PropTypes.arrayOf(hostPolicyInterface),
   query_results: PropTypes.arrayOf(hostQueryResult),
+  batteries: PropTypes.arrayOf(
+    PropTypes.shape({
+      cycle_count: PropTypes.number,
+      health: PropTypes.string,
+    })
+  ),
```

Comment:
FWIW I don't _think_ we need to `PropTypes` now that we're using TypeScript. I believe this is a remnant from JS that we never removed.

Reply:
we still use this in the targets interface which is used in `SelectTargetsInput.jsx`